src/test/java/spoon/test/template/testclasses/constructors/TemplateWithFieldsAndMethods.java

```diff
@@ -20,7 +20,7 @@ public TemplateWithFieldsAndMethods(String PARAM,
 	}
 
 	public String methodToBeInserted() {
-		return PARAM;
+		return "PARAM";

```

Comment:
This one bothers me: one of the main interest of Spoon template, is that they are java typed. Here using `return PARAM;` is right concerning the typing, but throws an exception as there's no field with the new name in the test. 
With the proposed solution, (i.e. string parameters are used for renaming fields/methods), I don't think we can keep the typing safety for String (like here). 
However we should provide an immediate feedback, when applying the template: I'd like to do it when a String parameter is used in this kind of case but it seems really complicated, maybe we can do it only when an error occured, instead of throwing an NPE, WDYT?